python/tskit/drawing.py

```diff
@@ -94,6 +94,12 @@ def linear_transform(self, y):
         return self.plot_min - (y - self.min_time) * y_scale
 
 
+class _svg_string(str):
```

Comment:
Can we use SVGString? As a class name it has to be capitalised.

Reply:
Done, and a bit of documentation added.